.github/workflows/changelog.yml

```diff
@@ -15,4 +15,4 @@ on:
 
 jobs:
   check_changelog:
-    uses: OpenTermsArchive/engine/.github/workflows/changelog.yml@main
+    uses: OpenTermsArchive/engine/.github/workflows/changelog.yml@v0.37.1
```

Comment:
I like that we version, but why stick to v0.37 instead of v1?

Reply:
Because the way changelog is handle change in the v1